code/modules/projectiles/gun.dm

```diff
@@ -189,9 +189,11 @@
 		for(var/obj/O in contents)
 			O.emp_act(severity)
 
-/obj/item/gun/afterattack_secondary(mob/living/victim, mob/living/user, params)
+/obj/item/gun/afterattack_secondary(mob/living/victim, mob/living/user, params, no_holdup = FALSE)
```

Comment:
Sorry, I wasn’t clear. Please make this a variable on the gun itself, not a parameter. A parameter can be useful, but not in this case. It means that everyone that doesn’t want a hold_up has to override the proc, when a simple no_holdup = TRUE would’ve sufficed.

Also not really wrong but calling it can_holdup prevents people having to work with double negatives (like no_holdup = FALSE), which helps a bit with readability